app/widget/slider/sliderlabel.cpp

```diff
@@ -54,9 +54,11 @@ void SliderLabel::mousePressEvent(QMouseEvent *e)
   if (e->button() == Qt::LeftButton) {
     if (e->modifiers() & Qt::AltModifier) {
       emit RequestReset();
-    } else {
+    } else  {
       emit LabelPressed();
     }
+  } else if (e->button() == Qt::RightButton) {
```

Comment:
Rather than catching `Qt::RightButton`, it would be better to set `setContextMenuPolicy(Qt::CustomContextMenu)` on the label. Then the label will automatically emit a `customContextMenuRequested` signal that you can connect to instead of a new signal.